torch/csrc/distributed/Placement.h

```diff
@@ -53,27 +56,18 @@ class Shard : public Placement {
   }
 };
 
-class StridedShard : public Shard {
+class StridedShard : public Placement {
  public:
+  std::int64_t dim;
   std::int64_t split_factor;
-  explicit StridedShard(std::int64_t dim, std::int64_t split_factor_)
-      : Shard(dim), split_factor(split_factor_) {}
+  explicit StridedShard(std::int64_t dim_, std::int64_t split_factor_)
```

Comment:
nit: why do we add underscore to `dim_` here? was it intentional in `Shard` or was that just a typo?